apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/RetryConformanceTestFixture.cs

```diff
@@ -56,6 +56,4 @@ private static string GetEnvironmentVariableOrDefault(string name, string defaul
         string value = Environment.GetEnvironmentVariable(name);
         return string.IsNullOrEmpty(value) ? defaultValue : value;
     }
-
-    internal static void SleepAfterBucketCreateDelete() => Thread.Sleep(2000);
```

Comment:
It probably should be in the fxiture, in case we ever use it in a diffirent test class.